cves/CVE-2016-2161.yml

```diff
@@ -186,9 +187,10 @@ lessons:
   native_wrappers:
     applies:
     note:
-  distrust_input:
-    applies:
-    note:
+  distrust_input: 
+    applies: true
+    note: |
+      The vulnerability  occurred because of a lack of proper checks on the memory allocated. 
```

Comment:
Amusingly enough given @matthewseaman 's burning desire for pipe-wrapping, I actually think this line should actually _not_ be piped :P